packages/auth/package.json

```diff
@@ -17,12 +16,14 @@
     "lint": "tslint -c tslint.json {src,__tests__}/**/*.{ts,tsx}",
     "lint:fix": "tslint --fix -c tslint.json {src,__tests__}/**/*.{ts,tsx}",
     "prebuild": "rimraf dist/",
-    "build": "rollup -c rollup.config.ts",
-    "build:debug": "rollup -c rollup.config.ts --configDebug",
+    "build": "npm run build:esm && npm run build:cjs && npm run build:umd",
```

Comment:
Build tasks look fine. But we're loosing `build:debug`, right?

Reply:
I don't think we were using it that much, right? Webpack doesn't have that debug mode by default as far as I know.

We have two modes now:
- By default, development mode is active.
- When using `NODE_ENV=production npm run build`, it will bundle our scripts for production. That's what we should put in our release script when we have it.